piccolo_admin/example.py

```diff
@@ -399,6 +403,14 @@ def validate_email(cls, v):
         return v
 
 
+class DownloadMoviesModel(BaseModel):
+    director_name: str
```

Comment:
Should we add a field validator, that director name must not be empty? Something like this.
```python
class DownloadMoviesModel(BaseModel):
    director_name: str

    @field_validator("director_name")
    def validate_director_name(cls, v):
        if v == "":
            raise ValueError("it cannot be empty")
        return v
```
Without that if an empty string is entered in the form, Piccolo admin downloads an empty `csv` file

Reply:
Good point